src/microbe_stage/Membrane.cs

```diff
@@ -294,6 +307,9 @@ private void SetMesh()
 
         engulfAnimationMeshInstance.Mesh = membraneData.GeneratedEngulfMesh;
         engulfAnimationMeshInstance.MaterialOverride = EngulfShaderMaterial;
+
+        mucocystAnimationMeshInstance.Mesh = membraneData.GeneratedEngulfMesh;
```

Comment:
If the mucocyst due to reverting mesh generation changes cannot use the same mesh as engulfing, then it can generate a separate mesh for it, but I'd obviously prefer if that wasn't required.

Reply:
I think it was easier to program as you can just turn on visibility or off.

Reply:
I don't doubt that, but unless it gets wide support any graphics change that impacts the engulfing animation I won't be accepting. So either the mucocyst effect needs to work with the exact same mesh (like it is done here) or it needs to generate a fully different mesh that would then be assigned here.